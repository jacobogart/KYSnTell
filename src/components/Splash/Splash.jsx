import React from 'react';
import { Link } from 'react-router-dom'

export const Splash = () => {
  return (
    <div className="Splash">
      <h4>Know your status</h4>
      <Link to="/kys" className="main-link" >
        KYS
      </Link>
      <h2 className="splash-n">n'</h2>
      <Link to="/tell" className="main-link" >
        Tell
      </Link>
      <h4>Tell your partners</h4>
    </div>
  )
}
