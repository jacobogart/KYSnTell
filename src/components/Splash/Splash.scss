.Splash{
  height: 70vh;
  width: 80%;
  text-align: center;
  margin-top: -200px;
}

.main-link,
.splash-n {
  color: white;
  font-weight: 800;
  font-size: 14vh;
  margin: 10px;
}

h4 {
  font-size: 4vh;
  margin: 15px;
}

.main-link {
  &:hover {
    cursor: pointer;
  }
}