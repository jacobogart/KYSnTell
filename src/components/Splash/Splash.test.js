import React from 'react';
import { shallow } from 'enzyme';
import { Splash } from './Splash';

describe('Splash', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(<Splash />)
    expect(wrapper).toMatchSnapshot();
  });
});