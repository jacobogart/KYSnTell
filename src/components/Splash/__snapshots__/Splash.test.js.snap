// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Splash should match snapshot 1`] = `
<div
  className="Splash"
>
  <h4>
    Know you status
  </h4>
  <Link
    className="main-link"
    to="/kys"
  >
    KYS
  </Link>
  <h2
    className="splash-n"
  >
    n'
  </h2>
  <Link
    className="main-link"
    to="/tell"
  >
    Tell
  </Link>
  <h4>
    Tell your partners
  </h4>
</div>
`;
